prow/jobs.md

```diff
@@ -70,7 +70,87 @@ that matches `trigger` will suffice. This is useful if you want to make one
 command that reruns all jobs. If unspecified, the default configuration makes
 `/test <job-name>` trigger the job.
 
-### Pod Utilities
+## Standard Triggering and Execution Behavior for Jobs
+
+When configuring jobs, it is necessary to keep in mind the set of rules Prow has
+for triggering jobs, the GitHub status contexts that those jobs provide, and the
+rules for protecting those contexts on branches.
+
+### Triggering Jobs
+#### Trigger Types
+
+`prow` will consider three different types of jobs that run on pull requests
+(presubmits):
+
+ 1. jobs that run unconditionally and automatically. All jobs that set
+     `always_run: true` fall into this set.
```

Comment:
should we explicitly call out the default?

Reply:
I believe he does in the next paragraph?